code/api/src/migrations/5-updateUsersTable.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.addColumn(
+        'users',
+        'description',
+        { type: Sequelize.TEXT }
+       ),
+      queryInterface.addColumn(
+        'users',
+        'image',
+        { type: Sequelize.TEXT,
+        defaultValue: '/images/stock/default_profile.jpg' }
+      ),
+      queryInterface.addColumn(
+        'users',
+        'shippingAddress',
+        { type: Sequelize.STRING }
+      )
+    ]);
+  },
+  down: function (queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.removeColumn(
```

Comment:
Thank you for going over this and explaining that removeColumn is the ability to update columns in the future.